docs/implementations.rst

```diff
@@ -65,3 +65,13 @@ Starlette
 
 Starlette is a minimalist ASGI library for writing against basic but powerful
 ``Request`` and ``Response`` classes. Supports HTTP.
+
+
+FastAPI
```

Comment:
The list is in alphabetical order and I presume we'd want to keep it that way, so move above Quart